rm-datasource/src/main/java/io/seata/rm/datasource/exec/LockRetryController.java

```diff
@@ -27,18 +30,17 @@
  * @author sharajava
  */
 public class LockRetryController {
-    private static int LOCK_RETRY_INTERNAL =
-        ConfigurationFactory.getInstance().getInt(ConfigurationKeys.CLIENT_LOCK_RETRY_INTERVAL, DEFAULT_CLIENT_LOCK_RETRY_INTERVAL);
-    private static int LOCK_RETRY_TIMES =
-        ConfigurationFactory.getInstance().getInt(ConfigurationKeys.CLIENT_LOCK_RETRY_TIMES, DEFAULT_CLIENT_LOCK_RETRY_TIMES);
```

Comment:
Keep this constants. Returns it in the get methods, if there is no customized config.